test/test_helpers/sphinx.rb

```diff
@@ -37,13 +36,15 @@ def index_for(model)
 
       def disable_search_jobs!
         ThinkingSphinx::Test::JOB_CLASSES.each do |clazz|
-          clazz.any_instance.stubs(:perform)
+          clazz.alias_method :_perform_original, :perform unless clazz.respond_to?(:_perform_original)
+          clazz.define_method(:perform) { |*, **| }
         end
       end
 
       def enable_search_jobs!
         ThinkingSphinx::Test::JOB_CLASSES.each do |clazz|
-          clazz.any_instance.unstub(:perform)
+          clazz.remove_method :perform
```

Comment:
I don't think it's a good idea to remove the method. If you do, then you can't restore it afterwards.